bears/vcs/git/GitCommitBear.py

```diff
@@ -31,6 +31,18 @@ class GitCommitBear(GlobalBear):
     }
     CONCATENATION_KEYWORDS = [r',', r'\sand\s']
 
+    _nltk_data_downloaded = False
+
+    @classmethod
+    def setup_dependencies(cls):
```

Comment:
Seems about right. It would be great if you could investigate this. Otherwise we could check and download the nltk data in `GitCommitBear.check_imperative`